store/postgres/src/chain_head_listener.rs

```diff
@@ -92,7 +93,11 @@ impl ChainHeadUpdateListener {
                         "head_block_number" => &update.head_block_number,
                     );
 
-                    update_sender.broadcast(()).map_err(|_| ())
+                    futures03::future::ready(
+                        update_sender
+                            .broadcast(())
+                            .map_err(|_| panic!("no listeners for chain head updates")),
```

Comment:
This wasn't a panic before. Is it a good idea to introduce one now?

Reply:
It's wrong to panic here, I've fixed it to ignore the error as it used to.